pkg/middlewares/aws_ebs_encryption_by_default_reconciler.go

```diff
@@ -0,0 +1,65 @@
+package middlewares
+
+import (
+	"github.com/snyk/driftctl/pkg/resource"
+	"github.com/snyk/driftctl/pkg/resource/aws"
+)
+
+// AwsEbsEncryptionByDefaultReconciler is a middleware that create remote equivalent
+// "aws_ebs_encryption_by_default" resources from state resources.
+// Since we don't have an ID for remote resources of this type.
```

Comment:
`AwsEbsEncryptionByDefaultReconciler is a middleware that either creates an 'aws_ebs_encryption_by_default' resource based on its equivalent state one just for the purpose of getting the Terraform custom Id, or removes the resource from our list of remote resources if it is not managed and is disabled.`